tests/integration/helpers/network.py

```diff
@@ -197,6 +197,8 @@ def __init__(
         self._docker_client = docker.from_env(version=os.environ.get("DOCKER_API_VERSION"))
 
         self._container = None
+        
+        self.image = "raftkeeper/raftkeeper-integration-helper"
```

Comment:
use a more intuitive name, such as `network_partition_image`

Reply:
Ok, i rename it from `raftkeeper-integration-helper` to  `raftkeeper-network-partition`